sdk/core/Azure.Core.Experimental/src/JsonData.cs

```diff
@@ -36,26 +36,69 @@ public class JsonData : IDynamicMetaObjectProvider, IEquatable<JsonData>
         private static readonly JsonSerializerOptions DefaultJsonSerializerOptions = new JsonSerializerOptions();
 
         /// <summary>
-        ///  Creates a new JsonData object which represents an JSON object with no properties.
+        ///  Creates a new JsonData object which represents the value of the given JsonDocument.
         /// </summary>
-        public JsonData() : this(Array.Empty<KeyValuePair<string, JsonData>>())
```

Comment:
@jsquire, what was the thinking behind removing the parameterless constructor?

Reply:
The type is intended to be immutable and enable consumption scenarios only.   There's no value in having a completely empty instance that you're unable to update... or is there?

Reply:
I see.  There is a test case for it -- `DefaultConstructorMakesEmptyObject` -- but this was likely miscategorized and should have been included with the mutable tests.

Reply:
There were a few that I think I needed to kill in the main tests file because they relied on mutability.